wildfly-mcp-server/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Generated by https://smithery.ai. See: https://smithery.ai/docs/config#dockerfile
+# Use Maven for building the application
+FROM maven:3.8.5-openjdk-21 AS builder
+
+# Set the working directory
+WORKDIR /app
+
+# Copy the pom.xml and source code
+COPY wildfly-mcp-server/pom.xml ./
```

Comment:
the pom.xml is in the current directory, should be COPY pom.xml ./, same for the src directory